src/frontend/src/utils/statusList/types.ts

```diff
@@ -3,10 +3,22 @@ import { SubmitButtonTextEnum } from '../../constant/constant';
 export type PageAccessControl = {
   [page: string]: {
     accessControl: boolean;
-    params?: any;
+    params?: any & PageParamProperties;
   };
 };
 
+// Add page param interfaces for each page.
+export type PageParamProperties = ReviewPageParams;
+
+export interface ReviewPageParams {
```

Comment:
in future iterations we can have a file/folder for the different pages and set these types there